pkg/network/ovn_kubernetes.go

```diff
@@ -242,12 +246,23 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 
 	controlPlaneReplicaCount, _ := strconv.Atoi(rcD.ControlPlane.Replicas)
 
-	err := wait.PollImmediate(5*time.Second, 280*time.Second, func() (bool, error) {
+	var heartBeat int
+
+	err := wait.PollImmediate(OVN_MASTER_DISCOVERY_POLL, OVN_MASTER_DISCOVERY_TIMEOUT, func() (bool, error) {
 		matchingLabels := &client.MatchingLabels{"node-role.kubernetes.io/master": ""}
 		if err := kubeClient.List(context.TODO(), masterNodeList, matchingLabels); err != nil {
 			return false, err
 		}
-		return len(masterNodeList.Items) != 0 && controlPlaneReplicaCount == len(masterNodeList.Items), nil
+		if len(masterNodeList.Items) != 0 && controlPlaneReplicaCount == len(masterNodeList.Items) {
+			return true, nil
+		}
+
+		heartBeat++
+		if heartBeat%3 == 0 {
+			klog.V(2).Infof("Wating to complete bootstrapping OVN: found (%d) master nodes out of (%d) expected: timing out in %d seconds",
```

Comment:
@JacobTanenbaum "Waiting to complete OVN bootstrap: found ..."

Reply:
done